more_itertools/more.py

```diff
@@ -4977,3 +4980,23 @@ def doublestarmap(func, iterable):
     """
     for item in iterable:
         yield func(**item)
+
+
+def _nth_prime_ub(n: int) -> float:
+    "Upper bound for the nth prime (counting from 1)."
+    # https://en.wikipedia.org/wiki/Prime-counting_function#Inequalities
+    return n * math.log(n * math.log(n)) if n >= 6 else 11.1
```

Comment:
There is a `from math import log` on line 25, so the `math.log()` can be replaced with just `log()` which  looks nicer.